apps/frontend/src/components/TrackerForm/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { colors } from "../colors";
+
+const Wrapper = styled.section`
+  background-color: purple;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+`;
+
+const Input = styled.input``;
+
+const Submit = styled.button`
+  background-color: ${colors["primary-100"]};
```

Comment:
I added themes to Styled you can use following
```suggestion
  background-color: ${props => props.theme.colors.primary['100']};
```

Reply:
works well